<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QrMainWindow</class>
 <widget class="QMainWindow" name="QrMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QrMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="userTextArea"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="submItButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>save current qr code</string>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQR_Code_Generator">
    <property name="title">
     <string>QR Code Generator</string>
    </property>
   </widget>
   <addaction name="menuQR_Code_Generator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
